docs/readmes/contributing/contribute_workflow.md

```diff
@@ -0,0 +1,189 @@
+---
+id: contribute_workflow
+title: Development Workflow
+hide_title: true
+---
+
+# Development Workflow
+
+This document covers the development workflow for contributing to the Magma project.
+
+## General workflow
+
+Magma follows the standard "fork and pull-request" development workflow. For more information on this workflow, consider the following
+
+- [Howto: fork and make pull request](https://guides.github.com/activities/forking/) [(visual version)](https://jarv.is/notes/how-to-pull-request-fork-github/)
+- [In-depth: fork and make pull request workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
+
+See the section at the end of this document for an opinionated version of these workflows.
```

Comment:
As opposed to saying 'See the section at the end of this document' I would rather add a direct reference to the section which will make the maintenance of this document easier as the link will either still be accurate after changes or you will know it needs updating.